modules/nf-core/gnu/sort/main.nf

```diff
@@ -0,0 +1,50 @@
+process GNU_SORT {
+    tag "${meta.id}"
+    label "process_low"
+
+    conda "conda-forge::coreutils=9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+    'ubuntu:20.04' }"
+
+    input:
+    tuple val(meta), path(file)
+
+    output:
+    tuple val(meta), file( "*.sorted" )   , emit: sorted
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args        = task.ext.args     ?: ''
+    def output      = task.ext.prefix   ?: "${meta.id}.txt"
+    output_file     = "${output}.sorted"
+    def VERSION     = "9.1"             // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+
+    """
+    cat ${file} | \\
+    sort ${args} > \\
+    ${output_file}
```

Comment:
```suggestion
    sort ${args} ${file} >  ${output_file}
```

I usually don't have a problem with superfluous cats, but here I think this looks nicer without it (and better on a single line).

Reply:
You're right, it's how it needed to be used in the previous pipeline but no more. Just tested and works exactly the same.